src/systems/super-spawner-system.js

```diff
@@ -3,7 +3,7 @@ import { ObjectContentOrigins } from "../object-types";
 
 // WARNING: This system mutates interaction system state!
 export class SuperSpawnerSystem {
-  maybeSpawn(state, entity, grabPath) {
+  maybeSpawn(state, grabPath) {
```

Comment:
Nothing changed in this file except that `entity` was being passed around these functions for no reason, so I stopped doing that.